### ISSUE TITLE
What is the expected loss after a successful pre-training on ImageNet dataset?

### ISSUE THREAD

**Issue by Youngseok0001** [facebookresearch/dino#60]
Thank you for making your code available. I am trying to re-implement your SSL technique (DINO) for other datasets. May I know what is considered a good loss value (or the lowest loss value you have observed) to stop the SSL pre-training on ImageNet dataset?

**Comment by mathildecaron31** [comment]
Hi @Youngseok0001 

You can take a look at the pre-training logs (`logs`) provided in the model zoo: https://github.com/facebookresearch/dino#pretrained-models
For example: https://dl.fbaipublicfiles.com/dino/dino_deitsmall16_pretrain/dino_deitsmall16_pretrain_log.txt

Hope that helps.

**Comment by zzzzzero** [comment]
> Hi @Youngseok0001
> 
> You can take a look at the pre-training logs (`logs`) provided in the model zoo: https://github.com/facebookresearch/dino#pretrained-models
> For example: https://dl.fbaipublicfiles.com/dino/dino_deitsmall16_pretrain/dino_deitsmall16_pretrain_log.txt
> 
> Hope that helps.
Thanks for sharing the train_log.
I wonder when the backbone is the transformer(vit/xcit), the train_loss will decrease at first, then increase, at last decrease, and the lowest loss value is not at the last epoch, while the backbone is resnet-50, the train_loss will decrease all the time.

 For image classification and image retrieval, when the backbone is the transformer, I wonder why not '" the lower loss value, the better performance"?
